rust_src/remacs-macros/README.md

```diff
@@ -0,0 +1,44 @@
+remacs-macros
+=============
+
+# Attributes
+
+## `lisp_fn`
+This macros creates the necessary functions and symbols automatically.
+It handles `MANY` functions and normal functions too (support for `UNEVALLED`
+needs to be done).
+
+It is used like this:
+
+```rust
+/// Return the same argument
+#[lisp_fn(name = "same", min = "1")]
```

Comment:
Do we support setting a maximum number of arguments in this macro? We probably should, so we can use it in this example.